cclib/parser/molcasparser.py

```diff
@@ -500,11 +517,11 @@ def extract(self, inputfile, line):
             line_max = next(inputfile).split()
             if not hasattr(self, 'geotargets'):
                 # The attribute geotargets is an array consisting of the following
-                # values: [Energy threshold, RMS Displacements threshold, RMS Gradient threshold, \
-                #          Max Displacements threshold, Max Gradient threshold].
-                self.geotargets = [self.energy_threshold, float(line_rms[4]), float(line_rms[8]), float(line_max[4]), float(line_max[8])]
+                # values: [Energy threshold, Max Gradient threshold, RMS Gradient threshold, \
+                #          Max Displacements threshold, RMS Displacements threshold].
+                self.geotargets = [self.energy_threshold, float(line_max[8]), float(line_rms[8]), float(line_max[4]), float(line_rms[4])]
```

Comment:
It would help readability if you could use tmp vars with intuitive names here, instead of line_max[8], line_rms[8], and do on.